### PR TITLE
Added ignoreDefaultLibraries API for Visual Studio 2010+ projects.

### LINKED ISSUE #89: Unable to set "Ignore Specific Default Libraries" in VS2010 project
There doesn't appear to be a way to set the "Ignore Specific Default Libraries" field in a Visual Studio project. (Located in Project Properties -> Linker -> Input)

The workaround is to use:
`linkoptions { '/NODEFAULTLIB:"libraryname"' }`

### PR DESCRIPTION
**Pull request by samsinsane**
Fixes #89

### CODE REVIEW

File: src/_premake_init.lua [hunk 1]
```diff
@@ -562,6 +562,13 @@
 	}
 
 	api.register {
+		name = "ignoreDefaultLibraries",
```
Comment (TurkeyMan) [inline src/_premake_init.lua]: api's are lower case.

Comment (samsinsane) [inline src/_premake_init.lua]: How did I manage to miss this... My bad, I'll fix this up.

File: tests/actions/vstudio/vc2010/test_link.lua [hunk 2]
```diff
@@ -460,3 +460,20 @@
 </Link>
 		]]
 	end
+
+--
+-- Test ignoring default libraries.
+--
+
+	function suite.ignoreDefaultLibraries()
+		ignoreDefaultLibraries { "lib1.lib", "lib2.lib" }
```
Comment (TurkeyMan) [inline tests/actions/vstudio/vc2010/test_link.lua]: This makes me wonder if should support lib names without `.lib` by auto-appending like the last issue?

File: tests/actions/vstudio/vc2010/test_link.lua [hunk 3]
```diff
@@ -460,3 +460,20 @@
 </Link>
 		]]
 	end
+
+--
+-- Test ignoring default libraries.
+--
+
+	function suite.ignoreDefaultLibraries()
+		ignoredefaultlibraries { "lib1.lib", "lib2.lib" }
```
Comment (TurkeyMan) [inline tests/actions/vstudio/vc2010/test_link.lua]: This makes me wonder if should support lib names without .lib by auto-appending like the last issue?

Comment (samsinsane) [inline tests/actions/vstudio/vc2010/test_link.lua]: That would be consistent, I'll add it in :)

**Comment by TurkeyMan** [comment]
I think this is good, but it'd be nice if `msc.lua` also gained this trick so makefiles work as well as vs projects.

**Comment by samsinsane** [comment]
I'll look into that as well :)

**Comment by TurkeyMan** [comment]
Thanks! Otherwise the disparity is basically just a bug waiting to be reported.

**Comment by TurkeyMan** [comment]
New api's need to be listed here too: https://github.com/premake/premake-core/wiki/Project-API
But that can happen after it's pulled.

**Comment by starkos** [comment]
What @TurkeyMan said re: casing and MSC support. Otherwise LGTM! Thanks!

**Comment by samsinsane** [comment]
@TurkeyMan @starkos I believe that's everything. Once it's pulled in, I'll add it into the wiki :)

**Comment by starkos** [comment]
Merged, thanks!

**Pull request merged**